src/graph/type_common.c

```diff
@@ -179,3 +175,29 @@ igraph_error_t igraph_edges(const igraph_t *graph, igraph_es_t eids, igraph_vect
 
     return IGRAPH_SUCCESS;
 }
+
+/**
+ * \function igraph_invalidate_cache
+ * \brief Invalidates the internal cache of an igraph graph
+ *
+ * </para><para>
+ * igraph graphs cache some basic properties about themselves in an internal
+ * data structure. This function invalidates the contents of the cache and
+ * forces a recalculation of the cached properties the next time they are
+ * needed.
+ *
+ * </para><para>
+ * You should not need to call this function during normal usage; however, we
+ * might ask you to call this function explicitly if we suspect that you are
+ * running into a bug in igraph's cache handling. A tell-tale sign of an invalid
+ * cache entry is that the result of a cached igraph function (such as
+ * \ref igraph_is_dag() or \ref igraph_is_simple() ) is different before and
```

Comment:
Just curious: is the space in `igraph_is_siple() )` needed for the doc tools to work correctly?